public/_posts/_updates/web-team/yearn-web-engineering-update-2022-02-21/es.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: "Actualización de ingeniería web: 21 de febrero 2022"
+categories: [Updates]
+image:
+  src: ./logo.png
+  width: 1256
+  height: 1024
+author: f0xTheSin
+date: "2022-02-24"
+translator: RestlessMik3
+---
+
+Fuente original: [Web Engineering Update 21 Feb](https://yearnweb.substack.com/p/yearn-web-engineering-update-b04?utm_source=url)
+
+#### Semana del 21 de febrero 2022
+
+## **Resumen**
+
+Arbitrum se lanza oficialmente en Yearn. Actualmente, solo hay una vault que emplea una estrategia (tricrypto de curve). Hemos aprendido de los dolores de crecimiento por los que pasamos con Fantom para ofrecer una implementación más fluida; para nuestros usuarios, nuestros estrategas y nuestros ingenieros.
+
```

Comment:
missing a paragraph here: "Our team continues to expand as we add chains, strategies, and vaults. We’ve brought in additional project managers and product owners to help wrangle the work and keep our community engaged."

Reply:
done, thanks